main.py

```diff
@@ -240,4 +181,5 @@ def sampler():
         tts(text_speaker, req_text, filename)
 
 if __name__ == "__main__":
-    main()
+    print(voices)
+    tts(session_id="73782eef66d5ab64bf83342be9623375", text_speaker="en_uk_001", req_text="This is a test", filename="test.mp3", play=True)
```

Comment:
This seems to have been a dev test that accidentally slipped in?
Should probably be reverted @oscie57

Reply:
Oops! I'll remove the test and submit another PR. Thanks for noticing.